apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1246,11 +1254,42 @@ namespace MWPhysics
             mStandingCollisions.clear();
         }
 
+        if (mNumThreads > 1)
+        {
+            int currentThread = 0;
+            std::vector<std::thread> threads(mNumThreads);
+            for(auto&& thread : threads)
+            {
+                thread = std::thread(&PhysicsSystem::solveTask, this, currentThread, mNumThreads, numSteps);
+                currentThread++;
+            }
+
+            for(auto&& thread : threads)
```

Comment:
Here the managing thread will stand idle while other threads are engaged in work. It is possible to generate `mNumThreads-1` of additional threads and do calculation as well in this thread